Makefile.build

```diff
@@ -70,6 +70,29 @@ include Makefile.ide      ## provides the 'coqide' rule
 include Makefile.install
 include Makefile.dev      ## provides the 'printers' and 'revision' rules
 
+###########################################################################
+# Adding missing pieces of information not discovered by ocamldep
+# due to the fact that:
+# - plugins/micromega/micromega_plugin.ml
+# - plugins/micromega/micromega_plugin.mli
+# are generated (and not yet present when we run "ocamldep").
+###########################################################################
+
+plugins/micromega/micromega_plugin.cmo : plugins/micromega/micromega.cmo
+plugins/micromega/micromega_plugin.cmx : plugins/micromega/micromega.cmx
+
+plugins/micromega/certificate.cmo plugins/micromega/coq_micromega.cmo plugins/micromega/csdpcert.cmo plugins/micromega/mfourier.cmo plugins/micromega/mutils.cmo plugins/micromega/polynomial.cmo : plugins/micromega/micromega.cmo
+
+plugins/micromega/certificate.cmx plugins/micromega/coq_micromega.cmx plugins/micromega/csdpcert.cmx plugins/micromega/mfourier.cmx plugins/micromega/mutils.cmx plugins/micromega/polynomial.cmx : plugins/micromega/micromega.cmx
+
+plugins/micromega/micromega.cmx plugins/micromega/micromega.cmo : plugins/micromega/micromega.cmi
+plugins/micromega/micromega.cmi : plugins/micromega/micromega.mli
+
+plugins/micromega/micromega.mli plugins/micromega/micromega.ml : plugins/micromega/MExtraction.vo
+	@:
```

Comment:
Since this commit `0cd64f60b1057e4c3daa0523ed42b00c315cf136` reals (e.g. `Ranalysis5.vo`) depend on the micromega-plugin.

However, `MExtraction.vo` depens on reals (indirectly also on `Ranalysis5.vo`) so I no longer can do what I wanted to do.

I.e. I can't extract `plugins/micromega/micromega.ml` by compiling `plugins/micromega/MExtraction.vo` because I can do that only when `theories/Reals/Reals.vo` already exists and that can exist only when micromega plugin is already available and that can be compiled only if `plugins/micromega/micromega.ml` already is available which is what I want to generate.

Chicken and egg problem.

That's why Travis complained (there are error messages about dropped dependencies due to cycles).